src/input/Slider.module.css

```
@@ -0,0 +1,110 @@
+/*
+Copyright 2021 Hugo Hutri <hugo.hutri98@gmail.com>
```

```suggestion
Copyright 2022 Hugo Hutri <hugo.hutri98@gmail.com>
```

This should also have a Matrix.org licence since some of the code is copied I belive